scripts/item-descriptions.lua

```diff
@@ -0,0 +1,75 @@
+-- Holds custom descriptions for view-item-info
+-- By PeridexisErrant
+
+print("This is a content library; calling it does nothing.")
```

Comment:
[Here](https://github.com/lethosor/dfhack-scripts/tree/master/example) is an example. (I don't know for sure if save-specific scripts in subdirectories are supported, but it should be easy enough to change the example paths.)